AdobeStockImage/Model/GetImageList.php

```diff
@@ -58,6 +58,7 @@ public function __construct(
     }
 
     /**
+     * @api
```

Comment:
Let's keep only the interface API, the implementation should not be an API. What do you think? cc @sidolov